virt-v2v/cold/WORKSPACE

```
@@ -188,8 +188,11 @@ bazeldnf_dependencies()
 
 container_pull(
     name = "centos9",
-    # 'tag' is also supported, but digest is encouraged for reproducibility.
-    digest = "sha256:40b2accec6e7ce1aa799610bac4a68cc378ad3dc0090ed57e984a2b119d92bd8",  # built on 2023-05-01
+    # Here we prefer using tag over a digest. CentOS rebuilds images often and
+    # does not keep older images. That means we would need to update the digest
+    # regularly. Also the assumption about reproducible builds is dubious
+    # because of our use of container_run_and_extract() during build.
+    tag = "latest",
```

do we wish latest or stream9(they also updates it and as far i understood it was pointing to stream9)?
```suggestion
    tag = "stream9"
```

---

it better be aligned with the base image we use, as long as it's ubi9(-minimal) I'd say the tag should rather be `stream9`

---

actually, looking at quay.io/centos/centos, 'latest' has not been modified for 2 years..